src/SWP/Bundle/BridgeBundle/Resources/config/doctrine-orm/Group.orm.yml

```diff
@@ -0,0 +1,32 @@
+SWP\Component\Bridge\Model\Group:
```

Comment:
we should add indexes here. One for (package, deletedAt). Not sure about code? Its used in query filters?